src/main/resources/application.yaml

```diff
@@ -182,3 +190,5 @@ payment-mode:
       type: "BULK"
       endpoint: "bulk_connector_{MODE}-{dfspid}"
       debulkingDfspid: "lion"
+batch-authorization:
+  callback-url: "https://bulk-connector.sandbox.fynarfin.io/authorization/callback"
```

Comment:
use the service name here. with in the cluster it will be able to connect

Reply:
There is no service with this name... what should I do?

Reply:
done